experiments/standalone/Vegetation/no_vegetation.jl

```diff
@@ -0,0 +1,270 @@
+import SciMLBase
```

Comment:
Nothing urgent, but I think in the coming months we should consider and have a discussion about how we write those experiments. 

I think it will be neat to use ClimaLandSimulations - it is not great at the moment and might need a refactor, but the idea is have fewer dependencies and fewers lines of code when writing experiments, making it easier to customize simulations. We will also use ClimaDiagnostics and ClimaAnalysis in those experiments, making them more generalizable and user friendly? 

Having said that, I understand that writing everything explicitly has advantages for debugging and also "no black box" which can be useful. 

Nothing to do in this PR, but I think we should talk about it. (I think end users like Mitra/Renato may have a different perspective than software engineer like Gabriele, and having a discussion with everyone's input would be informative)

Reply:
I agree we should talk about it! definitely we need ClimaLandSimulations+ClimaDiagnostics+Analaysis for end users (especially for large global simulations), but having these explicit scripts for some items may be useful too. 
- maybe it is also a question of being more clear about what our different tasks/uses are, and renaming "experiments" to be more clear about what they are
- maybe the need for "experiments" as we have them now will go down with time, after we add more core features to the models, and ClimaLandSimulations can become more stable